app/controllers/groups/create.js

```diff
@@ -0,0 +1,48 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class extends Controller {
+
+  @service errorHandler;
+
+  groupName ='';
+
+  groupEvents = [];
+
+  @action
+  addEvent(event) {
+    if (!this.groupEvents.includes(event)) {
+      this.groupEvents.push(event);
+    }
+  }
+
+  @action
+  shareEvent() {}
+
+  @action
+  async submit() {
+    try {
+      this.loading = true;
+      const myGroup = this.store.createRecord('group', {
+        user   : this.authManager.currentUser,
+        name   : this.groupName,
+        events : this.groupEvents
+      });
```

Comment:
Create the group in route itself. See how it is done in video stream route for create and edit

Reply:
ok

Reply:
is it wrong to create it in controller?

Reply:
Then you have to do things this way and it won't automatically do things we discussed in call